flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/catalog/hive/client/HiveMetastoreClientWrapper.java

```diff
@@ -149,6 +149,11 @@ public void dropDatabase(String name, boolean deleteData, boolean ignoreIfNotExi
 		client.dropDatabase(name, deleteData, ignoreIfNotExists);
 	}
 
+	public void dropDatabase(String name, boolean deleteData, boolean ignoreIfNotExists, boolean cascade)
+			throws NoSuchObjectException, InvalidOperationException, MetaException, TException {
+		client.dropDatabase(name, deleteData, ignoreIfNotExists, cascade);
```

Comment:
We need to check the availability of this API in every supported Hive versions.

Reply:
Yes，I check it hive version's range in [1.0.0, 3.1.2], all of them support this method.